dev/how-to/deploy/orchestrated/offline-ansible.md

```diff
@@ -163,13 +167,15 @@ See [Edit the `inventory.ini` file to orchestrate the TiDB cluster](/dev/how-to/
 
 1. You do not need to run the playbook in `ansible-playbook local_prepare.yml`.
 
-2. You can use the `Report` button on the Grafana Dashboard to generate the PDF file. This function depends on the `fontconfig` package and English fonts. To use this function, download the offline installation package, upload it to the `grafana_servers` machine, and install it. This package includes `fontconfig` and `open-sans-fonts`, and only supports the CentOS 7 system.
+2. You can use the `Report` button on the Grafana Dashboard to generate a PDF file. This function depends on the `fontconfig` package and English fonts. To use this function, download the offline installation package, upload it to the `grafana_servers` machine, and install it. This package includes `fontconfig` and `open-sans-fonts`, and only supports the CentOS 7 system.
```

Comment:
This item can be deleted in dev and 3.0. It cannot be used after the grafana version is upgraded.

Reply:
Got it.

Reply:
Updated. PTAL again~ @liubo0127

Reply:
LGTM